puzzle/models/variant.py

```diff
@@ -158,14 +158,17 @@ def _set_variant_id(self, variant_id=None):
         """Set the variant id for this variant"""
         if not variant_id:
             variant_id = '_'.join([
-                self['CHROM'],
-                self['POS'],
-                self['REF'],
-                self['ALT']
+                self.CHROM,
+                str(self.POS),
```

Comment:
Why is this type casting semantically right/needed (position should be an integer, or)?